core/arch/decodelib.c

```diff
@@ -40,21 +40,21 @@
 #include "../globals.h"
 #include "instr.h"
 
-#ifdef UNIX
+# ifdef UNIX
```

Comment:
This is another one where the outer define is around the whole file.  Perhaps there should be a general exception for that, with the outer marked as `/* around entire file*/` or sthg?